src/components/Delegates/DelegatesTabs/DelegatesTabs.tsx

```diff
@@ -50,7 +58,7 @@ export default function DelegateTabs({ children }: { children: ReactNode }) {
   return (
     <Tabs
       className="max-w-full"
-      defaultValue={tabParam || "delegates"}
```

Comment:
I extracted the internal state of `Tabs` into a managed state in this (`DelegateTabs`) component because we need access to that state in this component. We render the layout picker based on its state (only when value = `delegates`)